users/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json, jwt, requests
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+from django.conf import settings
+
+from .models import User
+
+class KakaotalkSignInView(View):
+    def get(self, request):
+        try : 
+            ACCESS_TOKEN        = request.headers.get('Authorization')
+            profile_information = requests.get(
+                "https://kapi.kakao.com/v2/user/me",
+                headers = {'Authorization' : f'Bearer {ACCESS_TOKEN}'}
+            )
```

Comment:
requests 모듈을 사용하실 때에는 timeout 옵션을 반드시 설정해주셔야 합니다.
그 이유는 timeout option을 추가하지 않으면 서버의 응답을 계속해서 기다리고 있는 문제가 발생 합니다. timeout 옵션을 추가해 주세요.
```suggestion
            profile_information = requests.get(
                "https://kapi.kakao.com/v2/user/me",
                headers = {'Authorization' : f'Bearer {ACCESS_TOKEN}'},
                timeout=5
            )
```

Reply:
timeout option 추가 했습니다.